common/src/main/java/com/microsoft/hydralab/performance/inspectors/AndroidMemoryHprofInspector.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package com.microsoft.hydralab.performance.inspectors;
+
+import com.microsoft.hydralab.common.util.ShellUtils;
+import com.microsoft.hydralab.common.util.TimeUtils;
+import com.microsoft.hydralab.performance.PerformanceInspection;
+import com.microsoft.hydralab.performance.PerformanceInspectionResult;
+import com.microsoft.hydralab.performance.PerformanceInspector;
+import org.slf4j.Logger;
+import org.springframework.util.Assert;
+
+import java.io.File;
+
+public class AndroidMemoryHprofInspector implements PerformanceInspector  {
+
+    private static final String RAW_RESULT_FILE_NAME_FORMAT = "%s_%s_%s_memory.hprof";
+    private static final String HPROF_FILE_PREFIX = "/data/local/tmp/";
+
+
+    @Override
+    public PerformanceInspectionResult inspect(PerformanceInspection performanceInspection, Logger logger) {
+
+        File rawResultFolder = new File(performanceInspection.resultFolder, performanceInspection.appId);
+        Assert.isTrue(rawResultFolder.exists() || rawResultFolder.mkdir(), "rawResultFolder.mkdirs() failed in" + rawResultFolder.getAbsolutePath());
+        String tmpTime = TimeUtils.getTimestampForFilename();
+        String hprofFileName = String.format(RAW_RESULT_FILE_NAME_FORMAT, getClass().getSimpleName(), performanceInspection.appId, tmpTime);
```

Comment:
The file path of the raw_result_file is too long, which may exceed the [Windows Maximum Path Length](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry). Can we shorten the file path?